sdk/aqueduct/utils/serialization.py

```diff
@@ -178,7 +246,9 @@ def artifact_type_to_serialization_type(
     elif artifact_type == ArtifactType.BOOL or artifact_type == ArtifactType.NUMERIC:
         serialization_type = SerializationType.JSON
     elif artifact_type == ArtifactType.PICKLABLE:
-        serialization_type = SerializationType.PICKLE
+        serialization_type = (
+            SerializationType.PICKLE if derived_from_bson else SerializationType.PICKLE
```

Comment:
hmmmmmm what's the difference between whether `derived_from_bson` or not?

Reply:
I guess you wanted `BSON_PICKLE`?

Reply:
o yeah good catch! The mongo integration test would have definitely helped cover this.